backend/src/tests/userschema.test.ts

```diff
@@ -0,0 +1,74 @@
+import { createTestClient } from 'apollo-server-testing'
+import gql from 'graphql-tag'
+import { server } from '../index'
+
+const AUTHORIZE_WITH_GH = gql`
+  mutation authorizeWithGithub($code: String!) {
+    authorizeWithGithub(code: $code) {
+      user {
+        id
+        username
+        emails
+        gitHubId
+        gitHubLogin
+        gitHubEmail
+        gitHubReposUrl
+        gitHubToken
+      }
+      token
+    }
+  }
+`
+
+const ME = gql`
+  query {
+    me {
+      id
+      username
+      emails
+      gitHubId
+      gitHubLogin
+      gitHubEmail
+      gitHubReposUrl
+      gitHubToken
+    }
+  }
+`
+
+describe('User schema mutations', () => {
+
+  it('user can not authorize with GitHub without providing a code as parameter', async () => {
+    const { mutate } = createTestClient(server)
+
```

Comment:
Tried to figure out what the typescript error for this line was but couldn't. Seems #153 just disabled similar error for the entire testfile. Maybe just add it here aswell if there's no other cure.

Reply:
Have you tried deleting node_modules and reinstalling?

Reply:
`/* eslint-disable @typescript-eslint/unbound-method */` talking about this error?

Reply:
`Avoid referencing unbound methods which may cause unintentional scoping of `this`.eslint@typescript-eslint/unbound-method` persists even with new node_modules. It compiles but vscode mentions this as an issue.